### ISSUE TITLE
Please add a widget to display feature file count

### ISSUE THREAD

**Issue by bawarim** [reportportal/reportportal#551]
I would like to see widget for execution status based on number of feature files executed, for my cucumber framework reports. 
Also if this feature count could be displayed in launches as well.

**Comment by bawarim** [comment]
I want to view the status of the total feature file executed. E.g. if 15 feature files has been executed then i can see the graph where it shows the number of feature file failed & passed. 

Please see the attachment for reference. 
[Feature_Widget.docx](https://github.com/reportportal/reportportal/files/2763485/Feature_Widget.docx)

**Comment by evjlobanova** [comment]
@bawarim  
We are planning to introduce after migration new feature #259  Sub-launch view  where user can see dashboard with widgets based on test cases from one particular launch. 
So on this dashboard we can consider to add widget that shown a status of all suites in launch. Does it cover your need?
 
Could you please share with  nesting levels of your tests. You report your results : Launch= Scenario> Feature > Test > Steps, doesn`t you?

**Comment by bawarim** [comment]
Nesting levels of my test is
Launch= Feature > Scenario > Steps, if i run it Through StepReporter. In this case i can see the dashboard widget by executed steps, not by scenario & features.

Launch= Feature > Scenario, if i run it Through ScenarioReporter. In this case i can see the dashboard widget by executed scenarios only, not by features.

So if #259 contains the flexibility to view the widget based on the different nesting levels then it serves my purpose.

**Comment by evjlobanova** [comment]
@bawarim 
so could you please confirm:
*Actual flow*
you create  Overall statistics widget;
you choose Filter for this widget;
output:
![image](https://user-images.githubusercontent.com/30413511/51252288-7c2e7700-19ac-11e9-897a-e5478d048b26.png)

*Requested flow*
you create Overall statistics widget;
you choose Filter for this widget;
![image](https://user-images.githubusercontent.com/30413511/51252962-34105400-19ae-11e9-951b-bdd037543d5e.png)

**Comment by bawarim** [comment]
Correct. I need all graphs (based on scenario, steps & Feature) for a same launch.

**Comment by evjlobanova** [comment]
@bawarim 

We have removed numbers from the widget with the purpose not to clutter up a graph.  So now we do not have any plans to turm them back. But you can use Panel view mode for widgets with type: Overall statistics. You will see information in format:
![image](https://user-images.githubusercontent.com/30413511/51312972-89576e80-1a5d-11e9-9da7-efc81fba717f.png)

**Comment by bawarim** [comment]
OK. So is there any possibility that can we get the graph view also along with the donut and panel view for overall statistics?

**Comment by evjlobanova** [comment]
@bawarim 

I should clarify one more thing.
1.) which type of widget do you use now?

**Comment by evjlobanova** [comment]
1.) 
![image](https://user-images.githubusercontent.com/30413511/51314000-f835c700-1a5f-11e9-8fbd-4c26ec588d81.png)


2.)
![image](https://user-images.githubusercontent.com/30413511/51313970-e7855100-1a5f-11e9-9662-5516b37f6581.png)

**Comment by bawarim** [comment]
Yes, i used the above widget with Donut view.

**Comment by bawarim** [comment]
> 2.) The second thing I want to understand is:
> as you have said, Feature = Launch
> you have the following structure:
> Launch in report portal **Feature A** - failed:
> 
> ```
>           Scenario 1  - passed
>                               Step 1 - passed
>                               Step 2 - passed
>                               Step 3 - passed
>            Scenario  2 - failed
>                               Step 1-failed
>                               Step 2 – skipped
>            Scenario  3 - failed
>                               Step 1-failed
>                               Step 2 – skipped
> ```
> Launch in report portal **Feature B** - passed:
> 
> ```
>           Scenario 1  - passed
>                               Step 1 - passed
>                               Step 2 - passed
> ```
> You need both graph:
> 1.) based on FEATURES from filter
> 
> ![image](https://user-images.githubusercontent.com/30413511/51314925-143a6800-1a62-11e9-9173-ce69aa59ff7a.png)
> 
> 2.) based on SCENARIOS for one launch
> ![image](https://user-images.githubusercontent.com/30413511/51315061-59f73080-1a62-11e9-8fb0-ece3c8e0317d.png)
> 
> 3.) based on STEPS for one launch
> ![image](https://user-images.githubusercontent.com/30413511/51315637-a3944b00-1a63-11e9-9cbf-cbf4fbb7bc34.png)
> 
> Am I right?

**Comment by bawarim** [comment]
No, Launch doesn't equals to feature. In a single launch i can have multiple features, like Feature A, Feature B Feature C and so on. 
Apart from this the graph which you have shared, is the exact thing which i need for a single launch.

**Comment by bawarim** [comment]
> - does panel view work for you?

Yes, panel view gives the information. But we generally used to share the data in pie chart view to the customer.

**Comment by evjlobanova** [comment]
@bawarim got it! Thank you for reply. We will consider this for implementation as part of Sub-launch dashboard functionality.

**Comment by dlitmano** [comment]
When you plan to release it?
